admin/RELEASE.md

```diff
@@ -55,6 +55,7 @@ the existing content.
     ```
 * Vet the **admin/** folders for any removed hidden files (Action deploy scripts
   *do not remove these*)
+  * git diff --name-status origin/master admin/
```

Comment:
@MGatner By the way, what if there is a removed hidden file?

Reply:
Like one beginning with a period? I think Git should still report on those.

Reply:
Ohhh I see what you mean: what do we do if a hidden file is actually removed. I don't recall what I did last time but we can probably just go remove it on the target repo before release.

Reply:
Yeah, we would have to delete it directly in that repository. But it's a bit cumbersome.
I don't think there is any chance of hidden files being deleted at this point, though.